specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2018-10-01/examples/getattesteddata.json

```diff
@@ -0,0 +1,57 @@
+{
+    "parameters": {
+        "api-version": "2018-10-01",
+        "nonce": "1111111111",
```

Comment:
nonce is supposed to be a number and not string as per the schema definition.

Reply:
Corrected this. Nonce can be up to 32 alphanumeric characters.